tox.ini

```diff
@@ -22,6 +22,13 @@ passenv =
     GCS_PROJECT_ID
 commands =
     py.test --cov=scrapy --cov-report= {posargs:--durations=10 docs scrapy tests}
+    
+[testenv:typing]
+basepython = python3
+deps =
+    mypy==0.770
+commands =
+    mypy --config-file=setup.cfg {posargs:scrapy}
```

Comment:
What do you think about running it on tests as well?

Reply:
I was also thinking mypy is going to pick up setup.cfg automatically

Reply:
Done in f28da1b0d648a9ef518302061d48fea48404c464. Initially I was under the impression that `setup.cfg` would be picked up, the docs say so. However, for some reason it wasn't working for me, so I decided to go the explicit way. Now it does, so perhaps I was doing something wrong :shrug: